assets/revisions/rev_oxu8ghlvuqmh_update_subtraction_nicknames_created_at_and_file_names.py

```diff
@@ -0,0 +1,91 @@
+"""update subtraction nicknames, created_at and file names
+
+Revision ID: oxu8ghlvuqmh
+Date: 2024-11-21 18:03:09.177151
+
+"""
+
+import datetime
+
+import arrow
+from syrupy.matchers import path_type
+
+from virtool.migration import MigrationContext
+
+# Revision identifiers.
+name = "update subtraction nicknames created_at and file names"
+created_at = arrow.get("2024-11-21 18:03:09.177151")
+revision_id = "oxu8ghlvuqmh"
+
+alembic_down_revision = "b79c1e6cf56c"
+virtool_down_revision = None
+
+# Change this if an Alembic revision is required to run this migration.
+required_alembic_revision = None
+
+
+async def upgrade(ctx: MigrationContext):
+    await ctx.mongo.subtraction.update_many(
+        {"nickname": {"$exists": False}},
+        {"$set": {"nickname": ""}},
+    )
```

Comment:
Needs a transaction around it. Multi updates are not atomic.

Reply:
Same as above